cmd/notation/cert_gen.go

```diff
@@ -45,21 +46,21 @@ func generateTestCert(ctx *cli.Context) error {
 	fmt.Println("generated certificates expiring on", cert.NotAfter.Format(time.RFC3339))
 
 	// write private key
-	keyPath := config.KeyPath(name)
+	keyPath := dir.Path.Localkey(name + dir.KeyExtension)
```

Comment:
I feel passing in 2 vars: `name` and `extention` might be more clear than constructing a name with extention manually.

Reply:
Done. Thanks!